### PR TITLE
dysgu==1.6.4_version update

### LINKED ISSUE #5784: new module: dysgu=v1.6.4
Hi, I am trying to learn how to add this new module "`dysgu`" for analysing structural variant in short or long reads in  `nf-core/sarek`.
```[tasklist]
### Tasks
- [ ] Creating conda environment for dysgu.nf
- [ ] I am searching for a PR for this module
- [ ] Working on meta.yml and main.nf files
```

### PR DESCRIPTION
**Pull request by poddarharsh15**
## PR checklist

Closes  #5784 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

**Comment by maxulysse** [comment]
Why closing this PR?

**Comment by poddarharsh15** [comment]
> Why closing this PR?

when I did the PR I had the same problems as last time .devcontainer missing and manta folder present that's why i closed it:(

**Comment by poddarharsh15** [comment]
> Why closing this PR?

Just reopened it now, I was making some errors in cloning that why I had to close it  :(

**Comment by poddarharsh15** [comment]
> > Why closing this PR?
> 
> Just reopened it now, I was making some errors in cloning that why I had to close it :(

I have run several tests on my workstation all tests were pass now waiting for github confirmation:)

File: modules/nf-core/dysgu/meta.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      licence: ["GPL-3.0-or-later"]
+
+input:
+  # Only when we have meta
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1' ]`
+
```
Comment (famosab) [inline modules/nf-core/dysgu/meta.yml:24]:
```suggestion
```

File: modules/nf-core/dysgu/meta.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  - dysgu:
+      description: Structural variant caller for mapped sequencing data
+      homepage: https://github.com/kcleal/dysgu
+      documentation: https://github.com/kcleal/dysgu/blob/master/README.rst
+      tool_dev_url: https://github.com/kcleal/dysgu
+      doi: "10.1093/nar/gkac039"
+      licence: ["GPL-3.0-or-later"]
+
+input:
+  # Only when we have meta
```
Comment (famosab) [inline modules/nf-core/dysgu/meta.yml:18]:
```suggestion
```

File: modules/nf-core/dysgu/meta.yml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+      description: |
+        Groovy Map containing reference information
+        e.g. [ id:'genome' ]
+  - fai:
+      type: file
+      description: Genome reference FASTA index file
+      pattern: "*.{fa.fai,fasta.fai}"
+
+output:
+  #Only when we have meta
```
Comment (famosab) [inline modules/nf-core/dysgu/meta.yml:48]:
```suggestion
```

File: modules/nf-core/dysgu/meta.yml [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'sample1'  ]
```
Comment (famosab) [inline modules/nf-core/dysgu/meta.yml:53]:
```suggestion
        e.g. [ id:'sample1' ]
```

File: modules/nf-core/dysgu/meta.yml [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+      type: file
+      description: The index of the BCF/VCF file
+      pattern: "*.{vcf.gz.tbi}"
+  - versions:
+      type: file
+      description: File containing software versions
+      pattern: "versions.yml"
+
+authors:
+  - "@kcleal"
```
Comment (famosab) [inline modules/nf-core/dysgu/meta.yml:68]: Is that correct?

File: modules/nf-core/dysgu/meta.yml [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+  - "@famosab"
+  - "@poddarharsh15"
+maintainers:
+  - "@kcleal"
```
Comment (famosab) [inline modules/nf-core/dysgu/meta.yml:72]:
```suggestion
  - "@kcleal"
  - "@poddarharsh15"
```

**Review by famosab — APPROVED** [review]
Looks good to me, just left some minor comments :)

File: modules/nf-core/dysgu/meta.yml (hunk 5 above)
Comment (poddarharsh15) [reply modules/nf-core/dysgu/meta.yml:68]: Yes it is, you mean tag of the developer right?

Comment (famosab) [reply modules/nf-core/dysgu/meta.yml:65]: usually you do not put the developer of the tool here but just the people taking care of the tool within nf-core (which is you in that case)

Comment (poddarharsh15) [reply modules/nf-core/dysgu/meta.yml:65]: Sure I will update it :)

**Pull request merged**